### ISSUE TITLE
updating the program?

### ISSUE THREAD

**Issue by Duemellon** [smy20011/dreambooth-gui#8]
When changes come along do we just download & rerun the MSI? is there a place to check for upgrades built in?

**Comment by smy20011** [comment]
Yes, just rerun the MSI. Updating is possible but I just wait for [this](https://github.com/tauri-apps/tauri-action/issues/251) to be implemented/

**Comment by bbecausereasonss** [comment]
Can we please get local CKPT models to be implemented instead of running off the tokens through web
